go/vt/wrangler/migrater_test.go

```diff
@@ -902,34 +902,6 @@ func TestMigrateNoTableWildcards(t *testing.T) {
 	}
 }
 
-func TestShardMigrateTargetMatchesSource(t *testing.T) {
```

Comment:
There is no way of testing it?

Reply:
the test was for the error message, which is no longer being produced.

Reply:
I meant in general. The whole test was deleted and nothing was added.

Reply:
I remember why I didn't add a test. It's more relevant to add one when we add the shard validation (the TODO that I added). But this change is a pre-req for the other changes I have lined up.